### PR TITLE
Dso 606 ml flow deploy helm to cluster

### PR DESCRIPTION
**Pull request by pmittal-verve**
I confirm that I have added:

- [x] A comment to every Dockerfile with information about target repository and tag (version).
- [x] Filled in readme on dockerhub and in this repo about how to build the image if some extra steps should be done.

### CODE REVIEW

**Comment by pmittal-verve** [comment]
@Konstantinos-Chaitas-PubNative @annoyingllama Please review.

File: mlflow/Makefile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+VERSION=0.0.1
+DOCKER_REPO=pubnative/mlflow
+DOCKER_REPO_VERSION=$(DOCKER_REPO):$(VERSION)
```
Comment (Konstantinos-Chaitas-PubNative) [inline mlflow/Makefile:3]: In the version we can use the mlflow version plus the git commit

File: mlflow/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FROM --platform=linux/amd64 python:3.8.10 AS build
```
Comment (Konstantinos-Chaitas-PubNative) [inline mlflow/Dockerfile:1]: I think it will be better to use the official MLflow Docker image directly

https://mlflow.org/docs/latest/docker.html

**Review by Konstantinos-Chaitas-PubNative — CHANGES_REQUESTED** [review]

**Comment by pmittal-verve** [comment]
@Konstantinos-Chaitas-PubNative updated please approve.

**Review by Konstantinos-Chaitas-PubNative — APPROVED** [review]

**Pull request merged**